drafts/use-cases/0.intro.md

```diff
@@ -48,31 +48,31 @@ These would be:
         },
         "operations": {
             "@id": "hydra:operation",
-            "@type": "hydra:Operation"
+            "@type": "@id"
         },
         "method": {
             "@id": "hydra:method",
             "@type": "xsd:string"
         },
         "returns": {
             "@id": "hydra:returns",
-            "@type": "hydra:Class"
+            "@type": "@id"
```

Comment:
This, `expects`, `supportedClass`, and `property` should probably be `"@type": "@vocab"`.
I think in general, we should have the same definitions as in the official context that can be found at https://github.com/HydraCG/Specifications/blob/master/spec/latest/core/core.jsonld